upcloud-api.postman_collection.json

```diff
@@ -523,6 +523,41 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "List object storages",
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "{{authorization}}"
+					},
+					{
+						"key": "Accept",
+						"value": "application/json"
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"url": {
+					"raw": "https://api.upcloud.com/1.3/object-storage/",
+					"protocol": "https",
+					"host": [
+						"api",
+						"upcloud",
+						"com"
+					],
+					"path": [
+						"1.3",
+						"object-storage",
+						""
+					]
+				}
+			},
+			"response": []
```

Comment:
I don't know if we should keep updating this postman config file. I feel it's really not part of this pkg.

Reply:
I second this. Also too easy to forget to update, since it's not mandatory for the feature itself to function properly. Should we remove that file?

Reply:
maybe delete the changes and create a second PR to remove it?

Reply:
https://github.com/UpCloudLtd/upcloud-go-api/pull/72